sqlglot/transforms.py

```diff
@@ -628,11 +628,27 @@ def eliminate_full_outer_join(expression: exp.Expression) -> exp.Expression:
             expression_copy = expression.copy()
             expression.set("limit", None)
             index, full_outer_join = full_outer_joins[0]
+
+            tables = (expression.args["from"].alias_or_name, full_outer_join.alias_or_name)
+            join_conditions = full_outer_join.args.get("on") or exp.and_(
+                *[
+                    exp.column(col, tables[0]).eq(exp.column(col, tables[1]))
+                    for col in full_outer_join.args.get("using")
```

Comment:
Can we get one more test showcasing `USING(a, b) -> WHERE NOT EXISTS (SELECT 1 FROM t1.a = t2.a AND t1.b = t2.b`, so we can cover the `OR` here?

Reply:
Sure, I think I can simply add more joining columns in the `USING` clause in [the one I already added] to cover for that (https://github.com/tobymao/sqlglot/pull/4178/commits/0b1b86c48e821ba70a096dff03d828afea559d45#diff-c5dbbef7e9cbb2c64b384079070c3006398036d834774db5983a1e347b7747f8R762-R767)

Reply:
Let's duplicate that test case to be sure that both work properly, thank you!